src/extend.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-empty-object-type */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { expect } from 'vitest';
+
+import { allCustomMatcherWithAliases, type CustomMatcher } from './matcher.js';
```

Comment:
Personally I dislike to mix type imports with regular imports, so I'd prefer this:
```suggestion
import type { CustomMatcher } from './matcher.js';

import { allCustomMatcherWithAliases } from './matcher.js';
```